### ISSUE TITLE
iocage stop failed

### ISSUE THREAD

**Issue by thedix** [iocage/iocage#793]
In this commit c7b509bb664bf5162510e075c67994af13e08efc iocage stop is not working.
```root@www:~ # iocage stop test1
* Stopping test1
  + Executing prestop OK
  + Stopping services FAILED
ERROR:
Terminated


Please use --force flag to force stop jail
```
The problem is caused by checking stderr output at [this line](https://github.com/iocage/iocage/blob/97796dd617ad311e97ff6f55cb684abe0cad33cc/iocage_lib/ioc_stop.py#L122) which value is 'Terminated' and later is supposed to be error.
Have no idea how to fix it, may be error checking should be based on return code instead of sdterr.

**Comment by sonicaj** [comment]
Thank you for letting us know @thedix. This bug was introduced a few hours back by another PR. We will have a fix up for this ASAP

**Comment by sonicaj** [comment]
This issue has been fixed by #792. Thanks again @thedix
